infrastructure/tooling/src/build/tasks/generic-worker-image.js

```diff
@@ -0,0 +1,156 @@
+const appRootDir = require('app-root-dir');
+const {
+  dockerPull,
+  dockerImages,
+  dockerRegistryCheck,
+  ensureTask,
+  dockerPush,
+  execCommand,
+} = require('../../utils');
+const path = require('path');
+
+/**
+ * This builds generic worker docker image containing all tools required to run generic worker
+ */
+module.exports = ({ tasks, baseDir, cmdOptions, credentials, logsDir }) => {
```

Comment:
This is a clone of `monoimage.js` .. Not sure if it's worth refactoring those build files to have some generic task generator, as those steps are mostly the same with regards to the steps: `monoimage`, `monoimage-dev`, `generic-worker`

Reply:
I think so - it was quite a big copy/paste :-/

Reply:
I will leave refactoring of docker building tasks out of this PR, as we've noticed that we also create on the fly most of the images, and that should probably be changed to static `Dockerfile`